stackrox/stackrox
# ROX-17763: Delete non-postgres code in Risk Acceptance UI

## Description

> deprecating our Postgres feature flag

Divide and conquer the residue of conditional code for Vulnerability Management.

## Checklist

## Testing Performed

1. `yarn lint` in ui
2. `yarn build` in ui
3. `yarn start` in ui

### Manual testing

1. Visit /main/vulnerability-management/risk-acceptance: see 3 GraphQL requests.

### Integration testing

Too bad, so sad; all tests are skipped in cypress/integration/vulnmanagement/riskAcceptance/observedCVEs.test.js
